Kiosk/KioskExperiences.cs

```diff
@@ -69,17 +103,99 @@ public class KioskExperience
         public KioskExperienceAttribute Attributes { get; set; }
     }
 
+    [Flags]
     public enum ExperienceType
     {
-        Kiosk,
-        Other
+        Automated = 1,
+        Guided = 2,
+        Business = 4,
+        IntelligentEdge = 8,
+        Fun = 16,
+        Experimental = 32,
+        Preview = 64
+    }
+
+    [Flags]
+    public enum TechnologyAreaType
+    {
+        Vision = 1,
+        Speech = 2,
+        Search = 8,
+        Language = 16,
+        Decision = 32
+    }
+
+    [Flags]
+    public enum TechnologyType
+    {
+        Face = 1,
+        Emotion = 2,
+        Vision = 4,
+        TextAnalytics = 8,
+        BingNews = 16,
+        BingImages = 32,
+        BingAutoSuggest = 64,
+        Bots = 128,
+        SpeechToText = 256,
+        CustomVision = 512,
+        QnA = 1024,
+        WinML = 2048,
+        TextToSpeech = 4096,
+        CognitiveSearch = 8192,
+        TranslatorText = 16384,
+        AnomalyDetector = 32768,
+        FormRecognizer = 131072
     }
 
     [AttributeUsage(AttributeTargets.Class, AllowMultiple = false, Inherited = false)]
     public class KioskExperienceAttribute : Attribute
     {
-        public string Title { get; set; }
+        public string DisplayName { get; set; }
+        public string Description { get; set; }
+        public string Id { get; set; }
         public string ImagePath { get; set; }
         public ExperienceType ExperienceType { get; set; }
+        public TechnologyAreaType TechnologyArea { get; set; }
+        public TechnologyType TechnologiesUsed { get; set; }
+        public bool IsPublic { get; set; } = true;
+        public string DateAdded { get; set; }
+        public string DateUpdated { get; set; }
+        public string UpdatedDescription { get; set; }
+        public int Rank { get; set; }
+        public bool IsSignedInRequired { get; set; } = false;
+        public string RequiredFile { get; set; }
+    }
+
+
+    [XmlRoot]
+    [XmlType]
+    public class RankedExperience
```

Comment:
You can remove this class and the one below (and associated code that was consuming it). No longer needed.